bokeh/models/annotations.py

```diff
@@ -338,6 +339,27 @@ class Label(Annotation):
     """)
 
 
+class Title(Label):
+    x = Override(default=0)
+
+    y = Override(default=0)
+
+    x_units = Override(default='screen')
+
+    y_units = Override(default='screen')
+
+    text_font_size = Override(default={'value' : '14pt'})
+
+    text_font_style = Override(default='bold')
+
+    def __init__(self, *args, **kwargs):
+        if len(args) == 1:
+            kwargs['text'] = [args[0]]
+        elif len(args) > 1:
+            raise ValueError("Too many positional arguments, expecting title text only")
+        super(Title, self).__init__(**kwargs)
```

Comment:
So you can do `Title("foo")` but if you do `Title(text="foo")` it will break in weird ways. My first reaction is to prefer to not subclass from `Label` for this reason. Tho possibly this would be acceptable:
```
def __init__(self, text, **kwargs): 
     # make sure 'text' key not in kwargs
     # make sure text param is a string
     kwargs['text'] = [text]
     super(Title, self).__init__(**kwargs)
```